app/models/admin_dashboard_data.rb

```diff
@@ -360,6 +361,12 @@ def s3_cdn_check
     end
   end
 
+  def translation_overrides_check
+    if TranslationOverride.exists?(status: %i[outdated invalid_interpolation_keys])
```

Comment:
Do you think we need an index here? We do query this in some (non-hot) paths, but I didn't want to add one prematurely given the table size has an upper bound that likely isn't that high. Happy to add it if someone thinks we need it.